### PR TITLE
Refactor simple changes

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Refactor simple changes

Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Review by joshuagl — APPROVED** [review]
nice cleanups, LGTM but I'll defer to @asraa and @laurentsimon as much more go savvy maintainers.

**Comment by laurentsimon** [comment]
Thanks @naveensrinivasan

**Pull request merged**
